exercises/darts/.meta/src/reference/java/Darts.java

```diff
@@ -0,0 +1,32 @@
+class Darts {
+
+    private static final int R_CIRCLE_OUTER = 10;
```

Comment:
Does `R` stand for radius? In which case I would prefer to use the whole word to make it clearer 🙂